src/chains/ethereum_1/examples/transaction.ts

```diff
@@ -109,17 +109,20 @@ const { env } = process
     await ropsten.connect()
 
     // ---> Sign and send ethereum transfer with compose Action - using generic (cross-chain) native chain transfer action
-    const transaction = await ropsten.new.Transaction(defaultEthTxOptions)
+    const transaction = (await ropsten.new.Transaction(defaultEthTxOptions)) as EthereumTransaction
```

Comment:
Add TODO comment to remove this type conversion after adding fees to main chain interface